docs/hooks/collections.mdx

```diff
@@ -209,6 +210,7 @@ import { CollectionAfterOperationHook } from 'payload/types'
 const afterOperationHook: CollectionAfterOperationHook = async ({
   args, // arguments passed into the operation
   operation, // name of the operation
+  req, // full express request
```

Comment:
Same `PayloadRequest` and JSDoc as above.

Reply:
.mdx only, not actual code :)